src/delegation/delegationController.ts

```diff
@@ -0,0 +1,288 @@
+import { IAccount } from "../accounts/interfaces";
+import { ITransactionOnNetwork } from "../interfaceOfNetwork";
+import { INetworkProvider } from "../networkProviders/interface";
+import { Transaction } from "../transaction";
+import { TransactionComputer } from "../transactionComputer";
+import { TransactionsFactoryConfig } from "../transactionsFactories";
+import { TransactionWatcher } from "../transactionWatcher";
+import { DelegationTransactionsFactory } from "./delegationTransactionsFactory";
+import { DelegationTransactionsOutcomeParser } from "./delegationTransactionsOutcomeParser";
+import * as inputs from "./resources";
```

Comment:
`inputs vs. resources` - what if we also had outputs? Can also be simply, `resources`.